discord/ui/select.py

```diff
@@ -436,6 +506,9 @@ def __init__(
             max_values=max_values,
             disabled=disabled,
             row=row,
+            default_values=MISSING
```

Comment:
It doesn't feel like `MISSING` should be used here. This ternary is also too long and should be split into a variable.